.eslintrc.js

```diff
@@ -62,6 +62,7 @@ module.exports = {
   },
   globals: {
     JSX: true,
+    AggregateError: false,
```

Comment:
This is to resolve eslint error `error 'AggregateError' is not defined`. Later versions of eslint have resolved the error.